README.md

```diff
@@ -55,6 +55,7 @@ Supported keys of the map are:
 ## License
 
 Copyright © 2016 Coda Hale
+Copyright © 2017 Sentry
```

Comment:
It's a first-class thing on GitHub. I can transfer this repo to you, then you can either work on it yourself or transfer it to the Sentry org yourself. The only thing I'd ask is to keep my name on the copyright line in the README (e.g. `Copyright 2017 Coda Hale, Sentry`).